### DISCUSSION TITLE
Como organizar o contato com um cliente?

### DISCUSSION THREAD

**Discussion by heliomsolivas** [frontendbr/forum#1186] [Discussão]
Ultimamente tenho falado muito com clientes na empresa onde eu trabalho, a princípio é tudo por e-mail e quando é necessário mostrar alguma coisa utilizo o Skype/Hangouts. O meu problema é que estou chegando a 200 e-mails só de replies e quando preciso achar alguma informação (tipo um link de um .psd que o cliente mandou, ou alguma alteração específica) fico "navegando" nos e-mails para encontrar.

Vocês possuem alguma estrutura de organização específica? Ou alguma ferramenta que facilite? Estou em busca de dicas para facilitar a minha organização.

**Comment by giovannism20** [comment]
Já tentou usar marcadores com os nomes e também pode linkar um drive com a conta, faça uma pasta também, o que acha ?

**Comment by drmcarvalho** [comment]
No Gitlab há um modulo de [Service Desk](https://docs.gitlab.com/ee/user/project/service_desk.html) que permite que você e seu time se conecte com partes externas através do e-mail gerado na plataforma do Gitlab. Basicamente, cada projeto que você cria é associado um e-mail para ele que pode ser usado para se comunicar com os usuários ou clientes como dito anteriormente.  E tem a finalidade de oferecer suporte.

O e-mail para Service Desk seria algo como:

`incoming+nomeusuario/nome-do-meu-repositorio@incoming.gitlab.com`

Você pode personalizar o modulo se quiser, e até ter um e-mail mais amigável que encaminhe para  este.

O primeiro e-mail que você recebe neste endereço é convertido numa _issue_ e você pode usar os recursos para categorizar ela e garantir a rastreabilidade, pode ser _labels_, _pesos  (valor de negócio)_, _milestone_, _datas_ e _notificações_. E as discussões que são geradas através do e-mail são convertidas em comentários na issue. 

Para mim esta forma de comunicação tem sido ótima, claro que você pode usar outras, mas como tudo fica centralizado no Gitlab, fica melhor para ter o controle.
